src/DependenciesPackager/CrossgenContext.cs

```diff
@@ -123,8 +123,8 @@ public void Crossgen(string outputPath)
             {
                 foreach (var asset in package.Assets)
                 {
-                    var subDir = CreateSubDirectory(outputPath, package, asset);
-                    var targetPath = Path.Combine(subDir.FullName, asset.FileName);
```

Comment:
Suppose there is a package "Newtonsoft.Json" with version "9.0.1-BETA" with a DLL "lib/net45/Newtonsoft.Json.dll".

Lowercased:

- <code>~/.nuget/packages/<b>newtonsoft.json</b></code>
- <code>~/.nuget/packages/newtonsoft.json/<b>9.0.1-beta</b></code>
- <code>~/.nuget/packages/newtonsoft.json/9.0.1-beta/<b>newtonsoft.json.9.0.1-beta.nupkg</b></code>
- <code>~/.nuget/packages/newtonsoft.json/9.0.1-beta/<b>newtonsoft.json.9.0.1-beta.nupkg.sha512</b></code>
- <code>~/.nuget/packages/newtonsoft.json/9.0.1-beta/<b>newtonsoft.json.nuspec</b></code>

Original case:

- <code>~/.nuget/packages/newtonsoft.json/9.0.1-beta/<b>lib/net45/Newtonsoft.Json.dll</b></code>